frontend/src/hooks/useSearchFilter.ts

```diff
@@ -1,21 +1,47 @@
+"use client";
+
```

Comment:
The name of this file might be better as `useSearchFilterAccordion` (if it's not also being used for opportunity status checkboxes - it would need to be refactored a bit for that) - but no pressing need to change the filename